src/app/query/bitcoin/runes/runes-outputs-by-address.query.ts

```diff
@@ -7,6 +7,8 @@ import { useCurrentNetwork } from '@app/store/networks/networks.selectors';
 import type { RunesOutputsByAddress } from '../bitcoin-client';
 import { useRunesEnabled } from './runes.hooks';
 
+const queryOptions = { staleTime: 1 * 60 * 1000 };
```

Comment:
Maybe we can share this across queries?

Reply:
This seems like quite a short stale time, no?